script/utils/diamondEMERGENCYPauseGitHub.sh

```
@@ -0,0 +1,154 @@
+# load helper functions
+source ./script/helperFunctions.sh
+
+# the number of attempts the script will max try to execute the pause transaction
+MAX_ATTEMPTS=10
+
+# Define function to handle each network operation
+function handleNetwork() {
+  echo ""
+  echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> start network $1 >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+  local NETWORK=$1
+  local PRIVATE_KEY=$2
```

Shellcheck

[warning] PRIVATE_KEY appears unused. Verify use (or export if used externally).

(SC2034)